src/main/resources/jp/ikedam/jenkins/plugins/scoringloadbalancer/ScoringLoadBalancer/help-simultaneousBuildsWorkaroundEnabled_ja.html

```diff
@@ -0,0 +1,8 @@
+<div>
+<p>
+TODO
```

Comment:
i don't know the best practices, but shouldn't we at lease use the english text here or remove the file to have the default help message?

Reply:
I don't know either. Who can help?

Reply:
What about this? Should I just remove these files?

Reply:
I removed them.

While testing, I noticed that even the other translation files (which already are translated into Japanese) like `help-enabled_ja.html` don't show seem to work:
![grafik](https://github.com/user-attachments/assets/1804d22e-a293-4292-ad73-7ac6e0cadc41)

Reply:
🤷‍♂️ I don't know enough about Jenkins translation to help. so simply don't add these empty Todo files.